content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -35,6 +35,7 @@ function relativeTime(timestamp) {
 
 const LAYOUT_VARIANTS = {
   basic: "Basic default layout (on by default in nightly)",
+  spocs: "A layout with all spocs shown",
```

Comment:
Still waiting on the layout for this.

Reply:
So it won't actually work. Also not sure what the endpoint ends up as, and "spocs" is only a guess.